src/components/Users/UserAvailabilityTab.tsx

```diff
@@ -73,22 +74,42 @@ export default function UserAvailabilityTab({
   const { hasPermission } = usePermissions();
   const { canViewSchedule } = getPermissions(hasPermission, permissions ?? []);
 
+  const from_date =
+    qParams.valid_from ??
+    format(startOfMonth(new Date()), "yyyy-MM-dd'T'HH:mm");
+  const to_date =
+    qParams.valid_to ?? format(endOfMonth(new Date()), "yyyy-MM-dd'T'HH:mm");
```

Comment:
follow js naming conventions please

edit: won't be needed as per next review comment